### PR TITLE
Update support.md to add support statement.

### PR DESCRIPTION
**Pull request by marrobi**
Closes issue #4

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-*Then remove this first heading from this SUPPORT.MD file before publishing your repo.*
-
 # Support
 
 ## How to file issues and get help  
 
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This project has been created to accelerate the development of Trusted Research Environments by Microsoft and it's partners while working with customers. This project does not have a dedicated team of maintainers outside of those who are working on an active engagements and as such issues will be responded to on a best efforts basis. No guarantees can be offered as to response times on issues, feature requests, or to the long term road map for the project.
```
Comment (christoferlof) [inline SUPPORT.md:5]: Nit: I believe `it's` should be `its`

**Review by christoferlof — CHANGES_REQUESTED** [review]
One minor fix required before I believe this is good to go. It's vs Its

File: SUPPORT.md (hunk 1 above)
Comment (marrobi) [reply SUPPORT.md]: You are correct. Your English is better than mine!

**Review by christoferlof — APPROVED** [review]

File: SUPPORT.md [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-*Then remove this first heading from this SUPPORT.MD file before publishing your repo.*
-
 # Support
 
 ## How to file issues and get help  
 
-This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
-feature request as a new Issue.
+This project has been created to accelerate the development of Trusted Research Environments by Microsoft and its partners while working with customers. This project does not have a dedicated team of maintainers outside of those who are working on an active engagements and as such issues will be responded to on a best efforts basis. No guarantees can be offered as to response times on issues, feature requests, or to the long term road map for the project.
```
Comment (ant0nsc) [inline SUPPORT.md:5]: Do we say somewhere that a Trusted Research Environment actually is?

Comment (marrobi) [reply SUPPORT.md:5]: Will be part of #2 and #3

**Review by cyberuna — APPROVED** [review]

**Pull request merged**